tests/model/test_model.py

```diff
@@ -1597,3 +1604,47 @@ def test_is_muted_topic(self, topic, is_muted, stream_dict, model):
         ]
         assert model.is_muted_topic(stream_id=topic[0],
                                     topic=topic[1]) == is_muted
+
+    @pytest.mark.parametrize('response', [
+        {
+            'emoji': {
+                '100': {
+                    'author_id': 5,
+                    'deactivated': False,
+                    'id': '100',
+                    'name': 'urwid',
+                    'source_url': '/user_avatars/1/emoji/images/1.png'
+                },
+                '20': {
+                    'author_id': 10,
+                    'deactivated': False,
+                    'id': '20',
+                    'name': 'snape',
+                    'source_url': '/user_avatars/2/emoji/images/2.png'
+                },
+                '3': {
+                    'author_id': 5,
+                    'deactivated': False,
+                    'id': '3',
+                    'name': 'dancing',
+                    'source_url': '/user_avatars/3/emoji/images/3.png'
+                },
+                '81': {
+                    'author_id': 5,
+                    'deactivated': True,
+                    'id': '81',
+                    'name': 'green_tick',
+                    'source_url': '/user_avatars/4/emoji/images/4.png'
+                },
```

Comment:
It's worth noting that this is deactivated in a comment - that's why this doesn't correlate to the `custom_emojis` fixture, presumably.